src/event_loop.cpp

```diff
@@ -62,7 +62,7 @@ EventLoop::~EventLoop() {
       rc = uv_loop_close(loop());
       if (rc != 0) {
         uv_print_all_handles(loop(), stderr);
-        assert(false && "Event loop still has pending handles");
+        LOG_ERROR("Event loop still has pending handles");
```

Comment:
This winds up printing "Event loop still has pending handles" in a (potentially) different location than the dump of handles to stderr.  Should we change the verbiage here to something like "Event loop still has pending handles; see status of event loop handles on stderr"?